sim/side.ts

```diff
@@ -592,7 +610,7 @@ export class Side {
 			return this.emitChoiceError(`Can't move: You can only ultra burst once per battle`);
 		}
 		let dynamax = (megaDynaOrZ === 'dynamax');
-		const canDynamax = this.activeRequest?.active[this.active.indexOf(pokemon)].canDynamax;
+		const canDynamax = this.activeRequest?.active[this.pokemon.indexOf(pokemon)].canDynamax;
```

Comment:
> You seem to be breaking the match between the indices of active and pokemon.

Oh, I understand now. Subactive Pokemon are retrofitted into ``activeRequest.active``, which is why this line had to be updated, and not because of some mismatch. You are making ``side.active`` an array of a single Pokemon, right?